src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      callback(number);
+
+      return calculator;
```

Comment:
you can return `this` here
this would be better because you don't need to remember the name of the variable `calculator` in order to access it
also, if you have to rename it, you will have to do it in one place